ocean/sediment/sed_tracer/100m/config_init.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<config case="init">
+
+	<add_executable source="model" dest="ocean_model"/>
+
+	<namelist name="namelist.ocean" mode="init">
+		<option name="config_ocean_run_mode">'init'</option>
+		<option name="config_init_configuration">'lock_exchange'</option>
+		<option name="config_vert_levels">-1</option>
+		<option name="config_write_cull_cell_mask">.true.</option>
+		<option name="config_use_debugTracers">.true.</option>
+                <option name="config_use_sedimentTracers">.true.</option>
+                <option name="config_lock_exchange_direction">'y'</option>
+                <option name="config_lock_exchange_vert_levels">50</option>
+		<option name="config_lock_exchange_bottom_depth">10.0</option>
+		<option name="config_lock_exchange_warm_temperature">30.0</option>
```

Comment:
Again, these should all be tabs.

Reply:
@xylar I fixed all the spaces-instead-of-tabs issues here manually.